k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java

```diff
@@ -473,6 +474,14 @@ private void setProgressBarPosition(int position) {
     }
   }
 
+  /**
+   *
+   * @return Selected thumbnail position in millis
+   */
+  public int getThumbnailPositionInMillis() {
```

Comment:
shouldn't it be a nullable Integer?

Reply:
I don't think so, I want it to be zero if the seekbar is not touched (the thumbnail would be the first frame of the video, as it was until now)